### ISSUE TITLE
Rename "Contents" to "On this page"

### ISSUE THREAD

**Issue by kmike** [pradyunsg/furo#60]
Proposal: currently the right sidebar is named "Contents". What do you think about changing the text to "On this page", like in https://stripe.com/docs/payments/payment-methods? I found "Contents" a bit ambiguous, as left sidebar also contains contents in a way, but for the whole docs.

**Comment by pradyunsg** [comment]
There's a lot of variations if this out in the interwebs. There's "Table of contents", "Contents", "On this Page" as well as no-title. TBH, I picked the one that looked good to me and ran with it.

There's reinforcing (and counter) examples for all these variants, as well as for "don't have anything". It also helps that this matches what I'd requested sphinx-book-theme to switch to. :)

I'm inclined to consider this one of the many aspects of this theme where "there were options and I picked one to enforce consistently".

Let me know if you have thoughts, and if you don't feel too strongly about this and this answer is satisfactory, feel free to close this issue! ^.^

Heads up: I'm on limited mobile data, coz I just moved so you might see me drop off for the next few days.
